apps/Cargo.toml

```diff
@@ -94,6 +94,7 @@ tendermint-proto = {git = "https://github.com/heliaxdev/tendermint-rs", branch =
 tendermint-rpc = {git = "https://github.com/heliaxdev/tendermint-rs", branch = "tomas/lowercase-node-id", features = ["http-client", "websocket-client"]}
 thiserror = "1.0.24"
 tokio = {version = "1.8.2", features = ["full"]}
+tokio-test = "0.4.2"
```

Comment:
We shouldn't use this crate in non-dev dependencies. We'll need to replace its usage too (more below)